app/gateway/database/postgres/account.go

```diff
@@ -16,12 +15,13 @@ type accountRepo struct {
 	db *pgxpool.Pool
 }
 
-func (t *accountRepo) GetWithCPF(ctx context.Context, document document.Document) (*account.Account, error) {
+func (t *accountRepo) GetWithCPF(ctx context.Context, document document.Document) (account.Account, error) {
 	const query string = "select * from accounts where document = $1"
```

Comment:
uma boa prática seria usar os campos expostos no select. 
Além de ser mais legível sobre quais dados estamos trabalhando, sendo mais fácil a manutenção do código. 
A ineficiencia pela indexaçao, uma vez selecionando os campos especificos, você tem um custo menor. 

Uma discussão legal sobre isso [aqui](https://pt.stackoverflow.com/questions/21583/por-que-usar-select-from-tabela-%C3%A9-ruim)
```suggestion
	const query string = `SELECT
	                       id,
	                       name,
	                       document,
	                       balance,
	                       secret,
	                       updated_at,
	                       created_at
	                     FROM 
	                       accounts 
	                     WHERE
	                       document = $1`
```

Reply:
Valeu pelo link! Então, realmente assim fica bem mais legível e explícito. Adicionei um issue sobre isso e em breve lanço um PR com essa correção. Obrigadoo :)